code/datums/storage/subtypes/medkits.dm

```diff
@@ -0,0 +1,129 @@
+///Regular medkit
+/datum/storage/medkit
+	open_sound = 'sound/items/handling/medkit/medkit_open.ogg'
+	open_sound_vary = TRUE
+	rustle_sound = 'sound/items/handling/medkit/medkit_rustle.ogg'
+	max_specific_storage = WEIGHT_CLASS_SMALL
+
+	///List of everything a medkit can hold
+	VAR_FINAL/static/list/obj/item/list_of_everything_medkits_can_hold = list(
+		//surgery tools
+		/obj/item/surgical_drapes,
+		/obj/item/scalpel,
+		/obj/item/circular_saw,
+		/obj/item/bonesetter,
+		/obj/item/surgicaldrill,
+		/obj/item/retractor,
+		/obj/item/cautery,
+		/obj/item/hemostat,
+		/obj/item/blood_filter,
+
+		//special tools
+		/obj/item/healthanalyzer,
+		/obj/item/dnainjector,
+		/obj/item/lazarus_injector,
+		/obj/item/implantcase,
+		/obj/item/implant,
+		/obj/item/implanter,
+
+		//stacks
+		/obj/item/stack/medical,
+		/obj/item/stack/sticky_tape,
+
+		//containers
+		/obj/item/reagent_containers/dropper,
+		/obj/item/reagent_containers/cup/beaker,
+		/obj/item/reagent_containers/cup/bottle,
+		/obj/item/reagent_containers/cup/tube,
+		/obj/item/reagent_containers/syringe,
+		/obj/item/reagent_containers/medigel,
+		/obj/item/reagent_containers/spray,
+		/obj/item/reagent_containers/hypospray,
+		/obj/item/reagent_containers/blood,
```

Comment:
You ended up removing all the applicator subtypes without adding the base applicator type

Reply:
crap